src/sonic-bgpcfgd/bgpcfgd/managers_device_global.py

```diff
@@ -228,6 +236,17 @@ def __extract_out_route_map_names(self, cmds):
                 route_map_names.add(result.group(1))
         return route_map_names
 
+    def get_chassis_tsa_status(self):
+        chassis_tsa_status = "false"
+        try:
+            ch = swsscommon.SonicV2Connector(use_unix_socket_path=False)
+            ch.connect(ch.CHASSIS_APP_DB, False)
+            chassis_tsa_status = ch.get(ch.CHASSIS_APP_DB, "BGP_DEVICE_GLOBAL|STATE", 'tsa_enabled')
+        except Exception:
+            pass
```

Comment:
Please log the message on exception. Helpful for debugging.

Reply:
Fixed. Thanks!